client/src/App.tsx

```diff
@@ -63,17 +63,22 @@ export function App() {
       setDeployResponse(output);
       toast.success(`Deployed ${imageName} but no route was created.`);
     }
-
   }
 
   const shadow = 0;
-
-  const [deployView, setDeployView] = React.useState(false);
+  
+  const [deployView, setDeployView] = React.useState(() =>{
+    return sessionStorage.getItem('deployViewState') === 'true';
```

Comment:
@fbricon localStorage seems to work between sessions. Should we use it? 
Would be also good to have button to return back to welcome page. WDYT?

Reply:
I think we could use localStorage to not show the welcome page every time, and indeed have some way to show it on demand afterwards.

If we want to keep selected image + deployment output, then sessionStorage would be best though.